src/sentry/api/endpoints/organization_sampling_project_rates.py

```diff
@@ -96,6 +108,14 @@ def put(self, request: Request, organization) -> Response:
         :auth: required
         """
 
+        self._check_feature(request, organization)
+
+        if organization.get_option("sentry:sampling_mode") != DynamicSamplingMode.PROJECT:
```

Comment:
Should this rather be checking for equality with the organization mode, to be explicit in case we ever add a third mode here?
```suggestion
        if organization.get_option("sentry:sampling_mode") == DynamicSamplingMode.ORGANIZATION:
```

Reply:
Doesn't that depend on the third mode? 
I think it's better to be explicit and more restrictive here and allow it only for Project mode, and then if the third mode should also allow it, we can add it also here in if-condition

Reply:
That depends on whether the third mode allows project rates or not, at the current stage it can be both. Sadly, Python doesn't have exhaustive checks and we better don't throw a `NotImplementError` in the production endpoint.